.env.dev.example

```diff
@@ -47,5 +47,5 @@ REDIS_PORT=6379
 REDIS_AUTH="myredissecret"
 
 LANGFUSE_WORKER_PASSWORD=mybasicauthsecret
-# openssl rand -base64 32 used only here
+# openssl rand hex 32 used only here
```

Comment:
conscious change within this PR?

Reply:
Yes conscious change as I wanted to bring up the issue. When I use the key generates with -base64 I get errors. Can you confirm that it's not a local issue?

Reply:
needs -hex flag, I'll edit and merge